### PR TITLE
Adding cron for daily tests and builds for meercode

### PR DESCRIPTION
**Pull request by dimpar**
The purpose of this PR is to set a cron job that would run tests and builds on a daily basis (at midnight) so the results can be displayed on Meercode dashboard.

### CODE REVIEW

File: .github/workflows/relay.yml [hunk 1]
```diff
@@ -77,13 +79,15 @@ jobs:
...
             && github.event_name != 'pull_request'
+            && github.event_name != 'schedule'
         uses: docker/login-action@v1
         with:
           registry: ${{ env.GCR_REGISTRY_URL }}
           username: _json_key
           password: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
       - name: Build and publish Docker Runtime Image
+        if: github.event_name != 'schedule'
```
Comment (michalinacienciala) [inline .github/workflows/relay.yml:90]: We could move the condition from here to the `push:` option. This way we would build the image and not push it (which would be consistent with the behaviour in other workflows).

File: .github/workflows/relay.yml [hunk 2]
```diff
@@ -106,6 +110,7 @@ jobs:
         # https://github.com/moby/buildkit/issues/1896
         # Without the change some jobs were failing with `no space left on device`
         name: Move cache
+        if: github.event_name != 'schedule'
```
Comment (michalinacienciala) [inline .github/workflows/relay.yml:113]: If we would build an image for `schedule event, this line would need to be removed.

File: .github/workflows/relay.yml (hunk 1 above)
Comment (dimpar) [reply .github/workflows/relay.yml]: Sure, done in 5f76e08

File: .github/workflows/relay.yml (hunk 2 above)
Comment (dimpar) [reply .github/workflows/relay.yml]: 5f76e08

**Review by michalinacienciala — APPROVED** [review]

**Pull request merged**